src/TesApi.Web/Management/ArmBatchQuotaProvider.cs

```diff
@@ -79,33 +78,33 @@ public async Task<BatchVmCoreQuota> GetVmCoreQuotaAsync(bool lowPriority, Cancel
     /// </summary>
     /// <returns></returns>
     public virtual async Task<AzureBatchAccountQuotas> GetBatchAccountQuotasAsync(CancellationToken cancellationToken)
-        => await appCache.GetOrCreateAsync(clientsFactory.BatchAccountInformation.ToString(), _1 => GetBatchAccountQuotasImplAsync(cancellationToken)); // TODO: Consider expiring the quota daily, because quota can be changed.
+        => await appCache.GetOrCreateAsync(clientsFactory.BatchAccountInformation.ToString(), entry =>
+        {
+            entry.AbsoluteExpirationRelativeToNow = TimeSpan.FromDays(1);
```

Comment:
what does this have to do with "Move away from ADAL"?

Reply:
You got me here. This was a drive-by to get rid of a simple TODO.